client/types/scheduledsearch.go

```diff
@@ -88,6 +94,22 @@ type ScheduledSearch struct {
 	DebugOutput     []byte        // output of the script if debugmode was enabled
 }
 
+type AlertMetadata struct {
+	Name     string            //name of the alert - REQUIRED
+	Category string            //category of the alert - OPTIONAL
+	Level    string            //level of the alert (info, warn, high, critical) - REQUIRED
+	Target   uuid.UUID         //The flow or script that will be fired if the query has results (may be blank)
+	Metadata map[string]string `json:",omitempty"` //set of additional free form keys that can be attached to an alert
+}
+
+// ReactiveInput represents a configured key that must be present in an reactive flow
+// The name indicates the name to be injected into the payload of a flow or the global environment for a script
+type ReactiveInput struct {
+	Required   bool   // if set to true then the reactive flow will not start if the value is missing
+	Name       string // name of the value to be injected
+	DebugValue string // temporary debug value to be injected into the payload when doing debug/testing runs
```

Comment:
By doing this, are we essentially saying that every ReactiveInput will always be a string?

Reply:
yes, i want to keep this simple.  The goal is to simplify the process of turning query into an alert